### PR TITLE
fixes out-dated URL for "Expressions allowed in guard clauses"

### PR DESCRIPTION
**Pull request by sgnl**
Noticed that this URL was out of date and found the URL.

### CODE REVIEW

File: bg/lessons/basics/control-structures.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ iex> case {1, 2, 3} do
 "Will match"
 ```
 
-Проветеве официалната документация за [Допустими изрази в предпазващи клаузи](http://elixir-lang.org/getting-started/case-cond-and-if.html#expressions-in-guard-clauses).
+Проветеве официалната документация за [Допустими изрази в предпазващи клаузи](https://hexdocs.pm/elixir/master/guards.html).
```
Comment (nscyclone) [inline bg/lessons/basics/control-structures.md]: Thanks, @sgnl!
I am not sure whether we should link to the `master` docs directly. Isn't [`stable`](https://hexdocs.pm/elixir/guards.html) a better choice?

Comment (doomspork) [reply bg/lessons/basics/control-structures.md]: I think that's a good idea @nscyclone 👍

**Comment by sgnl** [comment]
Thanks for the review. I've updated the URL to be:

`https://hexdocs.pm/elixir/guards.html#list-of-allowed-expressions`

**Comment by doomspork** [comment]
Thank you @sgnl and @nscyclone 👍

**Pull request merged**